### PR TITLE
implement multiply functionality

### PR DESCRIPTION
**Pull request by NaomiMunyiri**

### CODE REVIEW

**Review by MesfinGithinji — APPROVED** [review]

File: multiply_app/__init__.py
Comment (maxwelbarno) [inline multiply_app/__init__.py:1]: Add white space at the end of this file

File: multiply_app/routes.py
Comment (maxwelbarno) [inline multiply_app/routes.py:1]: Add whitespace

File: run.py
Comment (maxwelbarno) [inline run.py:1]: Add whitespace

File: multiply_app/__pycache__/multiply.cpython-310.pyc
Comment (maxwelbarno) [inline multiply_app/__pycache__/multiply.cpython-310.pyc:1]: Ignore this file. Ensure it is deleted from your repository

File: multiply_app/routes.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from . import multiplication_bp
+from flask import current_app as app, request, jsonify
+from .multiply import multiply
+
+@multiplication_bp.route('/multiply',methods=['GET'])
+def multiply_route():
+    num1=int(request.args.get('num1',0))
+    num2=int(request.args.get('num2',0))
+    result=multiply(num1,num2)
+    return jsonify({'result':result})
```
Comment (maxwelbarno) [inline multiply_app/routes.py]: Add whitespace

File: run.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    #blueprints
+    app.register_blueprint(multiplication_bp)
+
+    return app
+
+app=create_app()
+
+if __name__=='__main__':
+    app.run(debug=True)
```
Comment (maxwelbarno) [inline run.py]: Add whitespace

**Review by maxwelbarno — CHANGES_REQUESTED** [review]

File: run.py (hunk 2 above)
Comment (maxwelbarno) [reply run.py]: Hi @NaomiMunyiri, I have reviewed your code an left comments. Please work on them and let me know.
Happy Coding!

Comment (NaomiMunyiri) [reply run.py]: WIP

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+**/__pycache__/
```
Comment (maxwelbarno) [inline .gitignore]: Kindly add whitespace

**Review by maxwelbarno — CHANGES_REQUESTED** [review]

**Review by maxwelbarno — APPROVED** [review]

**Pull request merged**